### PR TITLE
IOOS stack

### PR DESCRIPTION
**Pull request by ocefpaf**
Moving more IOOS software to conda-forge.

### CODE REVIEW

File: recipes/pyugrid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        - numpy
+        - netcdf4
+
+test:
+    imports:
+        - pyugrid
+
+about:
+    home: https://github.com/pyugrid/pyugrid
+    license: BSD
```
Comment (ocefpaf) [inline recipes/pyugrid/meta.yaml]: Unspecified upstream. Seehttps://github.com/pyugrid/pyugrid/issues/140

File: recipes/pysgrid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        - numpy
+        - netcdf4
+
+test:
+    imports:
+        - pysgrid
+
+about:
+    home: https://github.com/sgrid/pysgrid
+    license: BSD
```
Comment (ocefpaf) [inline recipes/pysgrid/meta.yaml]: Unspecified. See https://github.com/sgrid/pysgrid/issues/90

**Comment by ocefpaf** [comment]
@jakirkham can you take a quick look and merge?

File: recipes/quantities/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+test:
+    imports:
+      - quantities
+      - quantities.constants
+      - quantities.tests
+      - quantities.units
+
+about:
+    home: http://pythonhosted.org//quantities/
+    license: Python-2.0
```
Comment (jakirkham) [inline recipes/quantities/meta.yaml]: Says BSD on PyPI.

Appears to be [BSD 3-Clause]( http://python-quantities.readthedocs.org/en/latest/user/license.html#license-agreement-for-quantities ).

Comment (ocefpaf) [inline recipes/quantities/meta.yaml]: But the site has: http://pythonhosted.org//quantities/user/license.html

File: recipes/pysgrid/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.1" %}
+
+package:
+    name: pysgrid
+    version: {{ version }}
+
+source:
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/sgrid/pysgrid/archive/v{{ version }}.tar.gz
+    md5: cec0fbcf461f42687dd3c3b5ab667e8f
```
Comment (jakirkham) [inline recipes/pysgrid/meta.yaml]: There is no PyPI version either so this is fine.

File: recipes/quantities/meta.yaml (hunk 3 above)
Comment (ocefpaf) [inline recipes/quantities/meta.yaml]: OK. The `readthedocs` sites is from 2012 and the `pythonhosted` is from 2011. Going with BSD-3-Clauses then. Thanks @jakirkham!

**Comment by jakirkham** [comment]
At least two packages are compiled (use Cython), which are `cell_tree2d` and `ciso`. Should we be running their test suites?

File: recipes/quantities/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+    imports:
+      - quantities
+      - quantities.constants
+      - quantities.tests
+      - quantities.units
+
+about:
+    home: http://pythonhosted.org//quantities/
```
Comment (jakirkham) [inline recipes/quantities/meta.yaml]: Nit. Extra `/`.

File: recipes/quantities/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/quantities/meta.yaml]: Interesting. Thanks @ocefpaf.

**Comment by ocefpaf** [comment]
> At least two packages are compiled (use Cython), which are `cell_tree2d` and `ciso`. Should we be running their test suites?

I don't think that is necessary. (As co-author of those packages I can say that their test suites are not ready.)

File: recipes/quantities/meta.yaml (hunk 5 above)
Comment (ocefpaf) [inline recipes/quantities/meta.yaml]: I took the opportunity to update that to point to http://python-quantities.readthedocs.org/en/latest/ instead.

**Comment by jakirkham** [comment]
Ok, I trust your judgement. LGTM assuming CI passes.

**Comment by jakirkham** [comment]
Thanks @ocefpaf.

**Comment by ocefpaf** [comment]
Thanks for the eagle eye review @jakirkham :smile:

**Comment by jakirkham** [comment]
What can I say. :smile:

**Pull request merged**